warehouse/query-core/src/main/java/datawave/query/postprocessing/tf/ContentFunctionArguments.java

```diff
@@ -107,7 +108,7 @@ public ContentFunctionArguments(Function f) throws ParseException {
                 }
             }
             
-            String nextArg = JexlASTHelper.dereference(args.get(currentArg++)).image.trim();
+            String nextArg = getValue(args.get(currentArg++));
```

Comment:
Fascinating.  I didn't realize negative numbers had this issue in jexl.  Do we have any other functions that process the arguments incorrectly as such?

Reply:
The geo functions which use northLat, southLat, eastLon, westLon have to have their arguments quoted in order to work.  That's the one that concerned me the most.  If you pass in the arguments unquoted it throws a NPE, which is good at least.  

I did not go through the effort of updating every function which accepts a number, but maybe that's something we should consider doing...